perfzero/scripts/setup_env.sh

```diff
@@ -12,3 +12,4 @@ export PERFZERO_BIGQUERY_TABLE_NAME="perfzero_dev.benchmark_summary"
 export PERFZERO_BIGQUERY_PROJECT_NAME="google.com:tensorflow-performance"
 export PERFZERO_EXECUTION_LABEL=perfzero_development
 export PERFZERO_ML_FRAMEWORK_BUILD_LABEL=nightly-gpu
+export PERFZERO_GCE_NVME_RAID=""
```

Comment:
If we are removing the arg I will need to know so we can setup fusion ahead of time.  We can update kokoro/fusion in advance.

Reply:
Sure. I will make sure to notice you and explain the backward incompatible interface change explicitly in the PR description.

Reply:
I updated the setup.py so that it keeps the previous behavior, i.e. setup disk if `--gcs_nvme_raid` is specified.